libs/ui/src/ListBox/ListBox.tsx

```diff
@@ -0,0 +1,15 @@
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { forwardRef } from 'react'
+
+import styles from './ListBox.module.css'
+import { ListBoxOption } from './ListBoxOption'
+
+type Props = PropsWithChildren<HTMLAttributes<HTMLUListElement>>
+
+export const ListBoxRoot = forwardRef(({ children, ...restProps }: Props, ref: ForwardedRef<HTMLUListElement>) => (
+  <ul {...restProps} ref={ref} className={styles.list}>
+    {children}
+  </ul>
+))
+
+export const ListBox = Object.assign(ListBoxRoot, { Option: ListBoxOption })
```

Comment:
I'm getting a typescript error on `.assign` that is does not exist on the ObjectConstructor.

Reply:
Hm, that's weird. `assign` has been in the spec for over 10 years 😅 Could this be a remnant from your NX PR?

Reply:
I think it was my editor that failed:p